4.3-pipelines/demonstrators/4.3.1-genotyping/README.md

```diff
@@ -1,3 +1,90 @@
 # Joint cohort genotyping demonstrator pipeline
 
-This is a stub page.
\ No newline at end of file
+This simple demonstrator pipeline follows the basic principles of the [common federation approach](../../README.md) adopted by CINECA WP4. The goal is to demonstrate how a simple metric (in this case, allele frequency) can be computed in a federated manner, without requiring ever collecting the raw (individual level) data in a central location.
+
+Step A (private step) reduces the individual level genotypes to dataset-specific allele numbers and counts, which are then exported and collected in a central location. Step B (meta-analysis step) then computes the final allele frequencies based on the results collected from step A.
+
+The instructions below demonstrate how the pipeline can be run on two separate datasets with two different reference genomes using three different execution environments. The example input files are provided. The test region used for the demonstrator is the ACE gene with the coordinates chr17:61554422-61575741 in GRCh37 and chr17:63477061-63498373 in GRCh38.
+
+## Local dependency installation
+
+```bash
+mkdir bin && cd bin
+# Nextflow
+wget -qO- https://get.nextflow.io | bash
+# Latest bcftools version (required for certain functionality)
+git clone --branch develop git://github.com/samtools/htslib.git
+cd htslib && git submodule update --init --recursive && make -j `nproc` && cd ..
+git clone --branch develop git://github.com/samtools/bcftools.git
+cd bcftools && autoheader && autoconf && ./configure && make -j `nproc` && cd ..
+git clone --branch develop git://github.com/samtools/samtools.git
+cd samtools && autoheader && autoconf && ./configure && make -j `nproc` && cd ..
+# Picard tools
+wget -q https://github.com/broadinstitute/picard/releases/download/2.24.2/picard.jar
+cd ..
+```
+
+## Step A1, raw data processing: GIAB
```

Comment:
Technically not raw data since you're grabbing bam files

Reply:
Fair point, renamed to “individual level data processing”